charts/seed-bootstrap/charts/etcd-druid/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Shouldn't we separate the `ServiceAccount` into its own file?

Reply:
Done.